wdl-lint/tests/lints/matching-param-meta/source.errors

```diff
@@ -1,3 +1,11 @@
+note[LineWidth]: line exceeds maximum width of 90
+  ┌─ tests/lints/matching-param-meta/source.wdl:1:1
+  │
+1 │ ## This is a test for checking for missing and extraneous entries in a `parameter_meta` section.
```

Comment:
Since this one is in a comment, can we split the comment into two lines so that the diagnostic doesn't fire?